src/styles/_catalog.scss

```diff
@@ -1,6 +1,22 @@
+// Variables
+$card-width: 200px;
+$card-gap: 48px;
+$screen-widths:
+  "488px" 2,
+  "768px" 3,
+  "1024px" 4;
+
+// Cataolog styling
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+
+  @each $screen-width, $card-count in $screen-widths {
+    @media (min-width: $screen-width) {
+      grid-template-columns: repeat($card-count, $card-width);
+    }
+  }
+  justify-content: center;
+  gap: $card-gap;
 
   max-width: 944px;
   margin: 0 auto;
```

Comment:
no margins on blocks

Reply:
Done!

Class page__catalog added to element